app/models/slack_url.rb

```diff
@@ -0,0 +1,12 @@
+class SlackUrl
+  def self.get_url(os)
+    options = Hash.new('slack unsupported os')
+    download_options = {
+      'Mac' => 'https://slack.com/ssb/download-osx',
+      'Windows' => 'https://slack.com/ssb/download-win',
+      'Linux' => 'https://downloads.slack-edge.com/linux_releases/slack-desktop-2.4.2-amd64.deb'
```

Comment:
Any reason why this is a specific .deb?

Also - we are saying this download link is for linux but this will only work on Debian based distros. Is there a way around that?